code/modules/mob/living/carbon/xenomorph/embryo.dm

```diff
@@ -54,12 +54,7 @@
 		affected_mob = null
 		return PROCESS_KILL
 
-	if(affected_mob.stat == DEAD)//DEAD CODE TO BE REMOVED
-		if(ishuman(affected_mob))
-			if(!HAS_TRAIT(affected_mob, TRAIT_UNDEFIBBABLE)) 
-				var/mob/living/carbon/xenomorph/larva/L = locate() in affected_mob
-				L?.initiate_burst(affected_mob)
-				return PROCESS_KILL
```

Comment:
Why remove this?

Reply:
It's an exact duplicate of the lines right under it, which get called if the !HAS_TRAIT check is false anyway.